dashboards-observability/server/adaptors/notebooks/default_backend.ts

```diff
@@ -224,12 +224,12 @@ export class DefaultBackend implements NotebookAdaptor {
    */
   deleteNote = async function (
     client: ILegacyScopedClusterClient,
-    noteId: string,
+    noteList: string,
```

Comment:
Is noteList supposed to be string[] ??

Reply:
Oh I see, it's converted to a string for the actual call

Reply:
yes the ids are passed as `id1,id2,id3` in http request, this should work as long as users don't manually construct an id with comma. I might improve it later to accept an array